src/igvfd/schemas/user.json

```diff
@@ -35,7 +35,7 @@
             "title": "Lab",
             "description": "Lab user is primarily associated with.",
             "comment": "See lab.json for available identifiers.",
-            "permission": "import-items",
```

Comment:
@gabdank I don't remember catching this.  Was the permissions working on ENCODEd at all with this - versus _

Reply:
@cricketsloan we discovered that ENCODE allows "permission" to be ANY string. I just suggest to be consistent with other places using import items (import_items)